### PR TITLE
chatbot created

### LINKED ISSUE #37: Chatbot for Homepage
I am thinking to go on with chatbot for homepage using JS, HTML  & CSS which will help to resolve basic issues faced by user.

### PR DESCRIPTION
**Pull request by NamanGokharu**
IMG_20180807_084042

I have successfully created chatbot with bunch of queries but official question which are more relevant are need to be added within array in popup_chatbot.js file.

### CODE REVIEW

**Review by shivshankar9 — APPROVED** [review]

File: home.css [hunk 1]
```diff
@@ -166,6 +166,132 @@ main button {
...
+  border:none;
+  outline: none;
+  cursor: pointer;
+}
+section{
+  max-width: 1000px;
+  margin: auto;
+  text-align: center;
+  padding: 0 1rem;
+}
```
Comment (Abirpal202049) [inline home.css]: Add class or id instead of element

File: index.html [hunk 2]
```diff
@@ -67,5 +67,29 @@ <h2>Tools</h2>
...
+    <!-- chatbot implementation-->
+    <span> 
+      <section>
+          <a class="float" id="float">
+              <img src="chatrobot.jpg" id="roboImg">
+          </a>
+          <div class="popup" style="width:500px;height:500px;">
+              <div class="textarea">
+                  <div class="roboside">
+                      <img src="chatrobot.jpg" class="avatar" alt="">
```
Comment (aakankshabhende) [inline index.html]: Can you please change the robot pic with the logo of student portal in the chat bot?

Comment (NamanGokharu) [reply index.html]: Ohkk, I will do above listed changes by evening! Hope it works. Can you provide the questions chart we actually want the chatbot to answer, so I can change it as well in JS.

**Review by NamanGokharu — COMMENTED** [review]
I have successfully done the changes, plz do review now!

File: index.html (hunk 2 above)
Comment (NamanGokharu) [reply index.html]: I have successfully done the changes, plz do review now!

File: home.css (hunk 1 above)
Comment (NamanGokharu) [reply home.css]: I have successfully done the changes, plz do review now!

**Review by aakankshabhende — APPROVED** [review]

**Comment by aakankshabhende** [comment]
Good Work :) @NamanGokharu

**Pull request merged**